packages/react/src/global-alert/GlobalAlert.tsx

```diff
@@ -1,63 +1,119 @@
+
+type GlobalAlertTone = 'info' | 'warning';
 
 export type GlobalAlertProps = PropsWithChildren<{
 	/** Function to be called when the 'Close' button is pressed. */
 	onDismiss?: MouseEventHandler<HTMLButtonElement>;
 	/** The title of the alert. */
 	title?: string;
+	/** The tone of the alert. */
+	tone?: GlobalAlertTone;
 }>;
 
-export function GlobalAlert({ children, onDismiss, title }: GlobalAlertProps) {
+const toneMap = {
+	info: {
+		bg: boxPalette.systemInfoMuted,
+		fg: boxPalette.systemInfo,
+		Icon: InfoFilledIcon,
+	},
+	warning: {
+		bg: boxPalette.systemWarningMuted,
+		fg: boxPalette.systemWarning,
+		Icon: WarningFilledIcon,
+	},
+};
+
+export function GlobalAlert({
+	children,
+	onDismiss,
+	title,
+	tone = 'warning',
+}: GlobalAlertProps) {
+	const { bg, Icon } = toneMap[tone];
+
 	return (
-		<section
-			aria-label={title}
-			css={{ backgroundColor: boxPalette.systemWarningMuted }}
-		>
-			<Content>
-				<Flex
-					flexDirection={['column', 'row']}
-					alignItems="flex-start"
-					justifyContent="space-between"
-					gap={[1, 1.5]}
-					paddingY={1}
-				>
-					<Flex flexDirection={['column', 'row']} gap={[0.5, 1]}>
+		<section aria-label={title} css={{ backgroundColor: bg }}>
+			<Flex
+				flexDirection="row"
+				css={{
+					position: 'relative',
+				}}
+			>
+				<GlobalAlertIconContainer tone={tone}>
+					<Icon aria-hidden="false" aria-label="Warning" color="inherit" />
+				</GlobalAlertIconContainer>
+
+				<Flex padding={1.5}>
+					<Stack gap={0.75} maxWidth={tokens.maxWidth.bodyText}>
+						{title ? (
+							<Heading
+								as="h2"
+								type="h3"
+								css={{
+									marginRight: mapSpacing(2),
+									[GLOBAL_ALERT_CLOSE_BUTTON_LABEL_VISIBLE_MQ]: {
+										marginRight: '0rem',
+									},
+								}}
+							>
+								{title}
+							</Heading>
+						) : null}
 						<Box
-							flexShrink={0}
-							css={{ color: boxPalette.systemWarning }}
-							paddingTop={title ? [0, 0.25] : undefined}
+							flexGrow={1}
+							css={
+								title
+									? null
+									: {
+											marginRight: mapSpacing(2),
+											[GLOBAL_ALERT_CLOSE_BUTTON_LABEL_VISIBLE_MQ]: {
+												marginRight: '0rem',
+											},
+									  }
+							}
 						>
-							<WarningFilledIcon
-								aria-hidden="false"
-								aria-label="Warning"
-								color="inherit"
-								css={{ display: 'block' }}
-							/>
+							{children}
 						</Box>
-						<Stack gap={0.5}>
-							{title ? (
-								<Heading as="h2" type="h3">
-									{title}
-								</Heading>
-							) : null}
-							<Box flexGrow={1}>{children}</Box>
-						</Stack>
-					</Flex>
-					{onDismiss ? (
-						<Button onClick={onDismiss} iconAfter={CloseIcon} variant="text">
-							Dismiss
-						</Button>
-					) : null}
+					</Stack>
 				</Flex>
-			</Content>
+				{onDismiss ? <GlobalAlertCloseButton onClick={onDismiss} /> : null}
+			</Flex>
 		</section>
 	);
 }
+
+const GlobalAlertIconContainer = ({
```

Comment:
Can we do

```
type GlobalAlertIconContainerProps = PropsWithChildren<{tone: GlobalAlertTone }>

function GlobalAlertIconContainer(props: GlobalAlertIconContainer) {}
```

Reply:
Currently we are mixing up `function Component` and `const Component = () => null` in the same file